app/uk/gov/hmrc/hmrcemailrenderer/templates/tdq/tdqFphReportNonCompliant.scala.html

```diff
@@ -0,0 +1,148 @@
+        </p>
+
+        <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">
+            Our logs show that the fraud prevention headers submitted for @reportParams.applicationName between
+            @reportParams.fromDate and @reportParams.toDate do not meet the specification on the HMRC Developer Hub.
+        </p>
+
+        <h2>What you need to do</h2>
+        <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">
+            Submitting header data for VAT (MTD) APIs is required by law. You'll find the fraud prevention specification
+            and the Test API on the HMRC Developer Hub.
+        </p>
+        <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">
+            To meet the specification, you need to:
+        </p>
+        <ol>
+            @if(reportParams.hasInvalidConnectionMethod) {
+                <li style="font-size: 19px; line-height: 1.315789474;">
+                    Go to the specification and select the correct connection method for your application.
+                    Your application has an invalid connection method in @reportParams.invalidConnectionMethodPercentage% of requests.
+                </li>
+            }
+            <li style="font-size: 19px; line-height: 1.315789474;">
+                Submit header data for every request made by your application.
+                @if(reportParams.hasAllHeadersMissing) {
+                    Your application is missing all of the header data in @reportParams.allHeadersMissingPercentage% of requests.
+                }
+            </li>
+            @if(reportParams.hasErrors) {
+                <li style="font-size: 19px; line-height: 1.315789474;">
+                    Correct the errors @if(reportParams.hasWarnings){and consider the warnings }shown in this report.
+                </li>
+            }
+            @if(reportParams.hasWarnings && !reportParams.hasErrors) {
+                <li style="font-size: 19px; line-height: 1.315789474;">
+                    Consider the warnings shown in this report.
+                </li>
+            }
+            <li style="font-size: 19px; line-height: 1.315789474;">
+                Use the Test API before you submit any headers. You need to check your data meets the specification and
+                is formatted correctly.
+            </li>
+        </ol>
+        <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">
+            If the Test API shows that you're submitting the correct headers, check if your end users are using an outdated
+            version of your application.
+        </p>
+        <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">
+            Most organisations are able to send all header data required for their connection method. If you’re unable to
```

Comment:
Wild apostrophe

Reply:
Copied and pasted from the Google Sheet :(
I've corrected this one and the others in this file and the `.txt` file too.